Walls/VoidstoneWall.cs

```diff
@@ -12,15 +12,31 @@ namespace CalamityMod.Walls
     public class VoidstoneWall : ModWall
     {
         internal static Texture2D GlowTexture;
+        internal static bool[,] GlowTexture_NonEmptyFrame = new bool[13, 5];
```

Comment:
Is there a way to instantiate this array so that it is based on the size of the sheet instead of being hard coded?